src/modules/CardsLogo.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <div class="cards">
+    <CardsList>
+      <slot />
+    </CardsList>
+  </div>
+</template>
```

Comment:
I'm not sure if I understand the intention of this CardsLogo file - can this be solved by just expanding the `CardsList` component instead of having this component as an intermediary?

Reply:
keeping it DRY since CardsList is also used in ApplyContent

Reply:
Yep, makes sense after the meeting we had – nice!